lib/test.js

```diff
@@ -53,6 +53,57 @@ function plan(count) {
 	this.plan(count, captureStack(this.plan));
 }
 
+function tryTest(title, fn, ...args) {
+	if (typeof title === 'function') {
+		args.unshift(fn);
+		fn = title;
+		title = undefined;
+	}
+
+	let committed = false;
+	let discarded = false;
+
+	this.addPendingAttemptAssertion();
+
+	const running = this
+		.forAttempt(title, t => fn(t, ...args))
+		.run();
+
+	return running.then(ret => {
+		const {passed, error, title, logs} = ret;
+		const errors = error ? Array.isArray(error) ? [...error] : [error] : [];
```

Comment:
We don't currently return an array of errors do we?

Reply:
Currently run() only returns one error, which I think is the first recorded error.
We can modify `saveFirstError(error)` to save array of errors, rather than one error.